lib/ddtrace/contrib/lograge/integration.rb

```diff
@@ -0,0 +1,44 @@
+require 'ddtrace/contrib/integration'
+require 'ddtrace/contrib/lograge/configuration/settings'
+require 'ddtrace/contrib/lograge/patcher'
+
+module Datadog
+  module Contrib
+    module Lograge
+      # Description of Lograge integration
+      class Integration
+        include Contrib::Integration
+
+        MINIMUM_VERSION = Gem::Version.new('0.11.0')
+
+        register_as :lograge
+
+        def self.version
+          Gem.loaded_specs['lograge'] && Gem.loaded_specs['lograge'].version
+        end
+
+        def self.loaded?
+          !defined?(::Lograge::LogSubscribers::Base).nil?
+        end
+
+        def self.compatible?
+          super && version >= MINIMUM_VERSION
+        end
+
+        # enabled by rails integration so should only auto instrument
+        # if detected that it is being used without rails
```

Comment:
Does it make sense for Lograge integration to work without Rails?

I don't see any Rails-specific code in `lib/ddtrace/contrib/lograge/instrumentation.rb`.

Reply:
> Does it make sense for Lograge integration to work without Rails?

Technically it has hard dependancies on `railties, activesupport, and actionpack` https://github.com/roidrage/lograge/blob/1729eab7956bb95c5992e4adab251e4f93ff9280/lograge.gemspec#L18-L20

So, this can probably just be `false`. i've updated

Reply:
Reading the project's Readme, I don't think it can work without Rails. 👍